ios/Handlers/LoginHandler.swift

```diff
@@ -10,13 +10,14 @@ class LoginHandler: NSObject {
     private var loginCompletion: ((ONGUserProfile?, Error?) -> Void)?
     private let pinAuthenticationEventEmitter = PinAuthenticationEventEmitter()
 
-    func handlePin(_ pin: String?) {
-        guard let pinChallenge = self.pinChallenge else { return }
+    func handlePin(_ pin: String?) -> Bool {
+        guard let pinChallenge = self.pinChallenge else { return false }
         guard let pin = pin else {
             pinChallenge.sender.cancel(pinChallenge)
-            return
+            return true
```

Comment:
No it should be true, but this will make more sense when I move it over to errors. Basically it should only return false when the pinChallenge does not exist.